File: src/android/com/eclipsesource/tabris/maps/MapValidator.java
package com.eclipsesource.tabris.maps;

import com.google.android.gms.maps.GoogleMap;

public class MapValidator {

  public static void validateGoogleMap( GoogleMap map, String message ) {
    if( map == null ) {
      throw new IllegalStateException( "Google Map is not yet ready. " + message );
    }
  }

}

File: src/android/com/eclipsesource/tabris/maps/MapValidator.kt
package com.eclipsesource.tabris.maps

import com.google.android.gms.maps.GoogleMap

object MapValidator {

  fun validateGoogleMap(map: GoogleMap?, message: String) {
    check(map != null) {
      "Google Map is not yet ready. $message"
    }
  }

}
